plugwise_usb/network/__init__.py

```
@@ -0,0 +1,569 @@
+                f"Invalid response message type ({response.__class__.__name__}) received, expected NodeJoinAvailableResponse"
+            )
+
+        mac = response.mac_decoded
+        _LOGGER.debug("node_join_available_message | sending NodeAddRequest for %s", mac)
+        try:
+            result = await self.register_node(mac)
+        except NodeError as exc:
+            raise NodeError(f"Unable to add Node ({mac}): {exc}") from exc
+        if result:
+            return True
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)